alerting/src/main/kotlin/com/amazon/opendistroforelasticsearch/alerting/model/action/ActionExecutionFrequency.kt

```diff
@@ -0,0 +1,183 @@
+/*
```

Comment:
UTs for `ActionExecutionFrequency` is missing, please open an issue and fix later.

Reply:
I added tests for the parse and streaming. Since this is mostly configuration I think the behavior as it relates to Action execute will be covered in MonitorRunner integration tests that we'll be adding for Aggregation Monitors.